app/renderer/js/utils/domain-util.js

```diff
@@ -225,7 +225,13 @@ class DomainUtil {
 	}
 
 	reloadDB() {
-		this.db = new JsonDB(app.getPath('userData') + '/domain.json', true, true);
+		const DOMAIN_JSON_PATH = path.join(app.getPath('userData'), '/domain.json');
```

Comment:
camelcase please i.e. `domainJsonPath`. Also, we need to do the same thing for `settings.json`, [here](https://github.com/zulip/zulip-electron/blob/master/app/renderer/js/utils/config-util.js#L50).